geode-dunit/src/main/java/org/apache/geode/internal/cache/functions/TestFunction.java

```diff
@@ -1032,6 +1032,11 @@ public boolean equals(Object obj) {
     return this.props.equals(function.getConfig());
   }
 
+  @Override
+  public int hashCode() {
```

Comment:
This implementation seems to violate the general rules of hashcode. It would use the same members as equals to generate a hashcode.

Reply:
Fixed. We now depend on the hashcode of the properties map,  if they have the same properties then they are equal and will have the same hashcode.

Reply:
I think the same issue exists on the other hashcodes added in this commit.